practica2023/src/main/java/com/endava/practica2023/Domain/EventType.java

```diff
@@ -0,0 +1,47 @@
+package com.endava.practica2023.Domain;
+
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Set;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@Builder
+@Table(name = "event_type")
+public class EventType {
+    @Id
+    @Column(name = "event_type_id")
+    private Integer eventTypeId;
+
+    @Column(name = "event_type_name")
+    private String eventTypeName;
+
+    @OneToMany(mappedBy = "eventType")
+    private Set<Eventss> events;
+
+    public Integer getEventTypeId() {
```

Comment:
Why didn't you use lombok to have as well getter and setters and not create them here?